module/collector-sqs/src/test/java/zipkin/module/collector/sqs/ZipkinSQSCollectorModuleTest.java

```diff
@@ -58,7 +55,7 @@ public void close() {
     if (context != null) context.close();
   }
 
-  @Test
+  @Test(expected = NoSuchBeanDefinitionException.class)
```

Comment:
I guess we'd usually use `assertThatThrownBy`

Reply:
imho overkill when only testing the exception class..